### PR TITLE
Added basic payment service

### PR DESCRIPTION
**Pull request by toshiks**

### CODE REVIEW

File: payment-service/models/base.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+from sqlalchemy.ext.declarative import declarative_base
```
Comment (BlooDCrescent) [inline payment-service/models/base.py]: @toshiks Can we move this thing into some common folder so others can reuse it or we need to copypaste it everywhere?

File: payment-service/dbworkers/SQLAlchemy.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+from sqlalchemy.engine import create_engine
```
Comment (BlooDCrescent) [inline payment-service/dbworkers/SQLAlchemy.py]: @toshiks Same thing as with model - can we share dbworkers?

File: payment-service/config.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+class Config:
+    APP_DIR = os.path.abspath(os.path.dirname(__file__))  # This directory
+    PROJECT_ROOT = os.path.abspath(os.path.join(APP_DIR, os.pardir))
+    SQLALCHEMY_DATABASE_URI = None
+    db_debug_mode = None
+
+
+class DevConfig(Config):
+    db_debug_mode = True
+    SQLALCHEMY_DATABASE_URI = "sqlite:///:memory:"
```
Comment (BlooDCrescent) [inline payment-service/config.py]: In-memory database for development? Why?

File: payment-service/app.py [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+
+@hug.extend_api()
+def apis():
+    return [api]
+
+
+@hug.local(skip_directives=False)
+def initialize(session: SQlAlchemySession):
+    info = PaymentInfo(order_id=10)
```
Comment (BlooDCrescent) [inline payment-service/app.py]: Dafuq?

File: payment-service/api.py [hunk 5]
```diff
@@ -0,0 +1,36 @@
+import hug
+
+from models.PaymentInfo import PaymentStatus, PaymentInfo
+from dbworkers.SQLAlchemySession import SQlAlchemySession
+from falcon.status_codes import HTTP_400, HTTP_200
+
+
+@hug.put("/api/orders/{order_id}/payment")
+def perform_payment(session: SQlAlchemySession, order_id: int):
+    new_payment_info = PaymentInfo(order_id=order_id)
```
Comment (BlooDCrescent) [inline payment-service/api.py]: Shouldn't we check if payment status is "authorized" as defined in the API? And depending on that we need to return smth?
Plus, I believe, we need to change order status as well so please add a mock call to order service

File: payment-service/models/base.py (hunk 1 above)
Comment (toshiks) [reply payment-service/models/base.py]: Nope, because every service has own code.

File: payment-service/config.py (hunk 3 above)
Comment (toshiks) [reply payment-service/config.py]: For this service it was convenient to have a new database after changes.

File: payment-service/app.py (hunk 4 above)
Comment (toshiks) [reply payment-service/app.py]: It's just test.

File: payment-service/config.py (hunk 3 above)
Comment (BlooDCrescent) [reply payment-service/config.py]: Why? It's a payment service. Shouldn't all changes be persistent in database from the beginning?

File: payment-service/app.py (hunk 4 above)
Comment (BlooDCrescent) [reply payment-service/app.py]: Ok

**Pull request merged**